clients/fossid-webapp/src/main/kotlin/Extensions.kt

```diff
@@ -261,3 +261,15 @@ suspend fun FossIdRestService.listPendingFiles(user: String, apiKey: String, sca
             "get_pending_files", SCAN_GROUP, user, apiKey, "scan_code" to scanCode
         )
     )
+
```

Comment:
Commit message nits: Use backticks instead of single quotes for nice rendering on GitHub.

"Add 'listIgnoreRules' function" -> "Add *a* 'listIgnoreRules' function" or "Add *the* 'listIgnoreRules' function"

"The 'listIgnoreRules' allows" -> "The 'listIgnoreRules' *function* allows"